api.php

```diff
@@ -1138,73 +1139,14 @@ public function run($userID, $permissionIsExplicit) {
 				$this->JSONResponse('A countryID is required.', '', false, ['countryID' => $countryID])
 			);
 
-		if ($limit > $limitAmount) {
-			throw new RequestException(
-				$this->JSONResponse('limit should not exceed 10000', '', false, ['limit' => $limit])
-			);
-		}
-
-		// Global Get all messages addressed to everyone
-		if ($countryID == 0) {
-			$where = "toCountryID = 0";
-		}
-
-		// Press Types
-		// Regular - Global and Private messaging allowed.
-		// PublicPressOnly - Only Global messaging allowed.
-		// NoPress - No messaging allowed.
-		// RulebookPress - No messaging allowed during 'Builds' and 'Retreats' phases.
-		else if ( 
```

Comment:
Wait, the code for controlling different press types used to be in getting messages rather than sending messages? Is this the big security hole?